### PR TITLE
Df row viz support

### PR DESCRIPTION
**Pull request by andypetrella**
cc @vidma since, again, you initiated this great one!

I just mention that the pivot chart was already available, the things were located in the post-df folder actually. however, were missing the Row's ToPoint and the simple Sample which I added.

Could you give that a spin and let me know what you think?

Note that I've updated the DataFrame example to show how PivotChart can be used ^^

thanks a lot man!

PS: gosh... I should have merged these machine learning stuffs before :-(

### CODE REVIEW

File: modules/common/src/main/post-df/magic.scala [hunk 1]
```diff
@@ -7,6 +7,22 @@ import notebook.front.widgets.isNumber
...
 trait ExtraSamplerImplicits {
   import SamplerImplicits.Sampler
+  /**
+   * count number of rows in a DataFrame, and do sampling if needed
+   */
+  implicit def SimpleDataFrameSampler = new Sampler[DataFrame] {
+    def apply(df: DataFrame, max: Int): DataFrame = {
+      val count = df.cache().count()
+      if (count > max) {
+        println("DF is larger than maxRows. Will use sampling.")
```
Comment (vidma) [inline modules/common/src/main/post-df/magic.scala]: btw, if you now how to properly print a message so it'd be shown to the end user,  go ahead and fix this.
> from within the sampler, `prinln` don't display any message, it seems.

**Comment by vidma** [comment]
looks :+1: (had no time to compile and run - I'm on holiday).

see the note on `println`.

**Comment by jarutis** [comment]
seems to work fine, go go go :+1: :)

**Comment by andypetrella** [comment]
'k let's merge it :-D

**Pull request merged**